zmk/templates/common/layouts.dtsi

```diff
@@ -0,0 +1,22 @@
+// See https://zmk.dev/docs/development/hardware-integration/physical-layouts
+// for how to configure this.
+
+#include <physical_layouts.dtsi>
+
+/ {
+    default_layout: default_layout {
+        compatible = "zmk,physical-layout";
+        display-name = "Default Layout";
+        transform = <&default_layout>;
+        kscan = <&kscan>;
+
+        // Edit this to define the positions and sizes of the keys
+        // https://zmk.dev/docs/development/hardware-integration/physical-layouts#optional-keys-property
+        keys  //                     w   h    x    y     rot    rx    ry
```

Comment:
Is it worth commenting this out since it is technically optional? I can see it either way.

Reply:
I think it's required in order to use Studio? I'm not sure how Studio behaves if this is not set.

I could add a comment saying you can remove it if not intending to support Studio though.

Reply:
Yeah, that sounds good.

Reply:
Updated. Does that look better to you?

Reply:
Yes, thanks!